sample/app/http-env-echo-no-loadbalancer.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
@thepwagner won't we need a jenkins pipeline capable of running `spacel-provision` (the suite) locked to master first? Or does this already float around somewhere?